phys/module_diagnostics_driver.F

```diff
@@ -493,6 +493,8 @@ SUBROUTINE diagnostics_driver ( grid, config_flags,                  &
                 ,QG_CURR=moist(ims,kms,jms,P_QG)                     &
                 ,QR_CURR=moist(ims,kms,jms,P_QR)                     &  !  gthompsn
                 ,NR_CURR=scalar(ims,kms,jms,P_QNR)                   &  !  gthompsn
+                ,NG_CURR=scalar(ims,kms,jms,P_QNG)                   &  ! THOMPSONGH
+                ,QVOLG_CURR=scalar(ims,kms,jms,P_QVOLG)              &  ! THOMPSONGH
```

Comment:
since NG and QVOLG are packaged with the new option, this would not be safe. I think THOMPSONGH needs a separate call to this routine.

Reply:
@dudhia should I separate the THOMPSONGH CASE statement from THOMPSON and THOMPSONAERO?

Reply:
@dudhia Does commit 0c99da3 address your request?

Reply:
This looks fine